rootstock-touch-install

```diff
@@ -41,10 +49,12 @@ do_shell()
 convert_android_img()
 {
 	if file $SYSIMG | grep -v ": Linux rev 1.0 ext4" >/dev/null; then
+	    echo "Converting sparse image to contiguious image"
 		simg2img $SYSIMG $WORKDIR/system.img.raw
 		mkdir $TMPMOUNT
 		mount -t ext4 -o loop $WORKDIR/system.img.raw $TMPMOUNT
-		make_ext4fs -l 120M $WORKDIR/system.img $TMPMOUNT >/dev/null 2>&1
+		IMAGE_SIZE=`wc -c < $SYSIMG`
+		make_ext4fs -l $IMAGE_SIZE $WORKDIR/system.img $TMPMOUNT
 		SYSIMAGE=$WORKDIR/system.img
```

Comment:
The IMAGE_SIZE fails for deb. I get a 

make_ext4fs -l 146982420 /tmp/rootstock-touch-install.olvH0/system.img /tmp/rootstock-touch-install.olvH0/tmpmount
Creating filesystem with parameters:
    Size: 146980864
    Block size: 4096
    Blocks per group: 32768
    Inodes per group: 4496
    Inode size: 256
    Journal blocks: 1024
    Label: 
    Blocks: 35884
    Block groups: 2
    Reserved block group size: 15
error: ext4_allocate_best_fit_partial: failed to allocate 484 blocks, out of space?

Not sure how to properly fix it, I'm working around by inserting: 
IMAGE_SIZE=$(( $IMAGE_SIZE * 2 ))